WcaOnRails/app/jobs/application_job.rb

```diff
@@ -1,4 +1,19 @@
 # frozen_string_literal: true
 
 class ApplicationJob < ActiveJob::Base
+  class DeferJob < StandardError
+    attr_reader :wait_time
+    def initialize(wait_time)
+      super("Delay the job by #{time} s")
```

Comment:
This might be gold plating/YAGNI', and I understand this is in part to fix a bug that we don't want to delay, so don't be shy about ignoring/deferring this, but:

Would it be useful to include info about _which_ job is being deferred? I imagine seeing this message somewhere and having no way to track which job it was.

You could pass the job itself here, and have a simple `.to_s` function in `ApplicationJob` (further legitimizing it).